references/data-converters-shared-flow/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "apigee-data-converters",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Please also remove the license field as all of DevRel is Apache2 licensed.

Reply:
ok removing